config/clusters/cloudbank/common.values.yaml

```diff
@@ -21,5 +21,5 @@ jupyterhub:
       name: quay.io/2i2c/cloudbank-data8-image
       tag: d2746e55a4ee
     nodeSelector:
-      # Put everything on the most appropriate nodepool for these users
-      cloud.google.com/gke-nodepool: nb-n2-highmem-4
+      # Put everything on the most appropriate instance type for these users
+      node.kubernetes.io/instance-type: n2-highmem-4
```

Comment:
@yuvipanda @GeorgianaElena @sgibson91 note that if we pin to a specific node pool name instance of an instance type, we can't do rolling updates of k8s versions by adding a new node pool and tainting nodes of the old node pool.

Due to this, we should avoid pinning to the node pool name, and instead pin to the instance type.